datafusion/ffi/README.md

```diff
@@ -17,21 +17,21 @@
   under the License.
 -->
 
-# `datafusion-ffi`: Apache DataFusion Foreign Function Interface
+# Apache DataFusion Foreign Function Interface
 
-This crate contains code to allow interoperability of Apache [DataFusion] with
-functions from other libraries and/or [DataFusion] versions using a stable
+This crate contains code to allow interoperability of [Apache DataFusion] with
+functions from other libraries and/or DataFusion versions using a stable
 interface.
 
 One of the limitations of the Rust programming language is that there is no
 stable [Rust ABI] (Application Binary Interface). If a library is compiled with
 one version of the Rust compiler and you attempt to use that library with a
 program compiled by a different Rust compiler, there is no guarantee that you
 can access the data structures. In order to share code between libraries loaded
-at runtime, you need to use Rust's [FFI](Foreign Function Interface (FFI)).
+at runtime, you need to use Rust's [FFI] (Foreign Function Interface (FFI)).
```

Comment:
this space fixes the link?

Reply:
Link was already working; spacing just for formatting